crates/viewer/re_selection_panel/src/item_title.rs

```diff
@@ -1,14 +1,25 @@
+use egui::WidgetText;
+
+use re_chunk::EntityPath;
 use re_data_ui::item_ui::{guess_instance_path_icon, guess_query_and_db_for_selected_entity};
-use re_ui::{icons, list_item, DesignTokens, SyntaxHighlighting as _, UiExt as _};
-use re_viewer_context::{contents_name_style, Item, SystemCommandSender as _, ViewerContext};
+use re_entity_db::InstancePath;
+use re_log_types::ComponentPath;
+use re_ui::{
+    icons,
+    syntax_highlighting::{InstanceInBrackets as InstanceWithBrackets, SyntaxHighlightedBuilder},
+    SyntaxHighlighting as _,
+};
+use re_viewer_context::{
+    contents_name_style, ContainerId, Contents, Item, SpaceViewId, ViewerContext,
+};
 use re_viewport_blueprint::ViewportBlueprint;
 
 #[must_use]
 pub struct ItemTitle {
-    name: egui::WidgetText,
-    hover: Option<String>,
-    icon: &'static re_ui::Icon,
-    label_style: Option<re_ui::LabelStyle>,
+    pub icon: &'static re_ui::Icon,
+    pub label: egui::WidgetText,
+    pub label_style: Option<re_ui::LabelStyle>,
+    pub tooltip: Option<WidgetText>,
 }
```

Comment:
For some reason, `ItemTitle` is needlessly exported by `re_selection_panel`. Can you please remove that line in `lib.rs`?